src/Connector.AzureDataLake/Connector/SqlDataWriter/ParquetSqlDataWriter.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+using CluedIn.Core;
+
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Logging;
+
+using Parquet;
+using Parquet.Schema;
+
+using ParquetRow = Parquet.Rows.Row;
+using ParquetTable = Parquet.Rows.Table;
+
+namespace CluedIn.Connector.AzureDataLake.Connector.SqlDataWriter;
+
+internal class ParquetSqlDataWriter : SqlDataWriterBase
+{
+    // From documentation, it's ambiguous whether we need at least 5k or 50k rows to be optimal
+    // TODO: Find recommendations or method to calculate row group threshold        
```

Comment:
Were you planning on doing this TODO?

Reply:
Created https://dev.azure.com/CluedIn-io/CluedIn/_workitems/edit/35784